src/RegistrationController.js

```diff
@@ -52,42 +61,61 @@ function (Okta, BaseLoginController, FormType, Enums, BaseSchema, SchemaFormFact
   return BaseLoginController.extend({
     className: 'registration',
     initialize: function() {
-      var Schema = BaseSchema.Model.extend({
-        url: 'api/v1/schema',
-        expand: ['schema']
+      var Schema = RegistrationSchema.Schema.extend({
+        url: 'api/v1/schema'
       });
       var schema = new Schema({
         schema: {
-          'properties':{
-            'firstname': {
+          'properties': {
+            'firstName': {
               'type': 'string',
-              'description': 'First name',
+              'description': 'First Name',
+              'default': 'Enter your first name',
```

Comment:
In which case is default displayed, since we already have description as placeholder? It is used for the errors?